### DISCUSSION TITLE
support for expo sdk 52 and next js 15

### DISCUSSION THREAD

**Discussion by erikcik** [t3-oss/create-t3-turbo#1274] [Ideas]
### Describe the feature you'd like to request

is there any plan to configure the template for the latest versions of expo and next js

### Describe the solution you'd like to see

update of the template

**Comment by 2-fly-4-ai** [comment]
> ### Describe the feature you'd like to request
> is there any plan to configure the template for the latest versions of expo and next js
> 
> ### Describe the solution you'd like to see
> update of the template
> 
> ### Additional information
> _No response_

Did you manage to do this? I am struggling atm with it. Everything works but the expo go app gets stuck at 100% and crashes if we try to open the debugger.

**Comment by juliusmarminge** [comment]
We went straight to Expo 53 so we can have react 19 everywhere, using multiple versions of React with hoisting is a pain... It's currently in canary, so just hold out until it's properly released if you don't want to roll a canary
